src/style.css

```
@@ -1 +1,58 @@
-/* Styles go here */
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
+header {
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  font-family: "Roboto";
```

```suggestion
  font-family: "Roboto", sans-serif;
```
no need to disable stylelint, just do what it wants)